#Expyriment 1.0.0 (Python 3.12.3), .xpe-file, coding: UTF-8
#date: Sun Sep 21 2025 12:06:52
#sha1: 0f3258
#modules: 
#display: size=(800, 600), window_mode=True, opengl=2
#os: uname_result(system='Linux', node='LeoSurface7', release='6.6.87.2-microsoft-standard-WSL2', version='#1 SMP PREEMPT_DYNAMIC Thu Jun  5 18:31:42 UTC 2025', machine='aarch64')
Time,Type,Event,Value,Detail,Detail2
143,stderr,received,'Traceback (most recent call last):'
143,stderr,received,'  File "/home/leo/cogsup-prog/CORE1/Week-2/Exercises/labeled_shapes.py", line 13, in <module>'
143,stderr,received,'    l1=stimuli.Line(0,50,line_width=5,colour=(255,234,8))'
143,stderr,received,'       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^'
143,stderr,received,'  File "/home/leo/projects/exp/.venv/lib/python3.12/site-packages/expyriment/stimuli/_line.py", line 50, in __init__'
156,stderr,received,'    self._start_point = list(start_point)'
156,stderr,received,'                        ^^^^^^^^^^^^^^^^^'
156,stderr,received,"TypeError: 'int' object is not iterable"
